### ISSUE TITLE
Reset

### ISSUE THREAD

**Issue by Christian-Uronarti** [urapadmin/kiosk#1197]
Hiya Lutz, 
not a software issue but an issue nonetheless. Can I reset Lottie's workstation? She apparently has not made any changes to her copy of the DB, has uploaded it and this is stopping her ws being synchronised (or rather, prepared)
Best,
CK

**Comment by urapadmin** [comment]
yes, whenever you want to. Just use the reset option under "more options" after you clicked on her workstation. 
Should you not see that option, let me know.

**Comment by urapadmin** [comment]
by the way: You still are listed as "pending invite" in github. Did my last invitations to github end up in your spam folder or some such?

**Comment by Christian-Uronarti** [comment]
umm- I contacted you via github --?? I definitely accepted your invitation
________________________________

**Comment by urapadmin** [comment]
anybody can open a ticket here without having been invited to begin with. That is not related. I'll kick you out and send you another invitation right away, please try once more.

**Comment by urapadmin** [comment]
yay! Now you are all set. 🎈
